src/languages/es.ts

```diff
@@ -1866,6 +1866,7 @@ export default {
             customTagName: 'Nombre de etiqueta personalizada',
             enableTag: 'Habilitar etiqueta',
             addTag: 'Añadir etiqueta',
+            editTag: 'Editar etiqueta',
```

Comment:
@waterim Did we confirm this translation?

Reply:
@waterim Bump

Reply:
This is correct!